README.md

```diff
@@ -58,6 +58,8 @@ Iceberg is built using Gradle with Java 1.8 or Java 11.
 * To invoke a build and run tests: `./gradlew build`
 * To skip tests: `./gradlew build -x test -x integrationTest`
 * To fix code style: `./gradlew spotlessApply`
+  The above command applies to default versions. 
+  Need to pass the non-default versions via `-DsparkVersions`, `-DflinkVersions`, `-DhiveVersions` argument
```

Comment:
you might want to mention an example as well: `./gradlew spotlessApply -DflinkVersions=1.13,1.14,1.15 -DsparkVersions=2.4,3.0,3.1,3.2,3.3 -DhiveVersions=2,3`.

Reply:
I intentionally didn't add the example thinking that we need to modify the `ReadMe.md` each time we change the version number.  (when a new version is added or the old version is removed)

But thinking more about it, maybe it is ok to add it. 

I have added it now.